.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Write release and finish
+
+on:
+  push:
+    tags:
+      - v*
+
+
+jobs:
+  finish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+          token: ${{ secrets.TOOLS_BOT_PAK }}
+
+      - name: Get associated PR
+        uses: helaili/github-graphql-action@2.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOOLS_BOT_PAK }}
+        with:
+          query: .github/queries/asssociated-pr.query.yml
+          outputFile: pr.json
+          owner: asfadmin
+          name: hyp3-rtc-gamma
+          sha: ${{ github.sha }}
+
+      - name: Export PR body
+        id: pr
+        run: |
+          PR_QUERY='.data.repository.commit.associatedPullRequests.edges[0].node.body'
+          PR_BODY=$(jq --raw-output "${PR_QUERY}"  pr.json)
+          echo "::set-output name=body::${PR_BODY}"
+
+      - name: Create Release
+        id: create_release
```

Comment:
I don't see this id being referenced anywhere, could drop it

Reply:
It's being used here: https://github.com/asfadmin/hyp3-rtc-gamma/pull/85/files/817c00f4062a3bee136e6b4391b51328dd2ceb26#diff-56c30d9224897b08c28075052d4595c5R44

Reply:
that's the id from the preceding `Export PR body` step, not the `Create Release` step?

Reply:
Your right -- it's in their suggested usage but it's not needed.